src/cdn/loadCKEditorCloud.ts

```diff
@@ -100,7 +100,15 @@ export function loadCKEditorCloud<Config extends CKEditorCloudConfig>(
 		loadedPlugins: combineCdnPluginsPacks( plugins ?? {} )
 	} );
 
-	return loadCKCdnResourcesPack( pack ) as Promise<CKEditorCloudResult<Config>>;
+	return loadCKCdnResourcesPack(
+		{
+			...pack,
+			htmlAttributes: {
+				crossorigin: 'anonymous',
```

Comment:
So it is now not possible to not include `crossorigin` attribute. I guess this is fine.

Reply:
It's possible, `anonymous` is default one but can be overridden by `injectedHtmlElementsAttributes` line below

Reply:
Are you sure? For me it seems that `crossorigin` will be always included and it can be overriden. But it is not possible to not include this attribute.

Reply:
You are right, I was thinking that setting `crossorigin: ''` will reset it to the default, but it turns out that it is not.